### PR TITLE
website: remove deprecated algolia config file

### PR DESCRIPTION
**Pull request by slumbering**
Signed-off-by: jffarge <jf@dagger.io>

### CODE REVIEW

File: website/env.example [hunk 1]
```diff
@@ -1,7 +1 @@
-# https://www.algolia.com/apps/XSSC1LRN4S/explorer/browse/Dagger_docs
-APPLICATION_ID=XSSC1LRN4S
-# Copy your "Write API KEY" from https://www.algolia.com/account/api-keys/all?applicationId=XSSC1LRN4S
-API_KEY=
-# This will not work on an ARM Mac, the algolia/docsearch-scraper image was only built for linux 64bit
-# I am using a Docker Engine on Linux here:
-DOCKER_HOST=ssh://192.168.1.22
+REACT_APP_AMPLITUDE_ID=
```
Comment (gerhard) [inline website/env.example:1]: @slumbering Can we add a link as a comment above where we get this value from? E.g.

```
# https://analytics.amplitude.com/blocklayer/settings/projects/333523/general
REACT_APP_AMPLITUDE_ID=
```

**Review by gerhard — CHANGES_REQUESTED** [review]
Almost there:
- I will push one change shortly
- One change for you @slumbering

File: website/env.example (hunk 1 above)
Comment (gerhard) [reply website/env.example:1]: Do you know what link we should use? I'm not sure which one you had in mind.

Comment (slumbering) [reply website/env.example:1]: Actually I tried to find where the amplitude api key but couldn't find it. It was Friday, I'll see that on Monday.

Comment (gerhard) [reply website/env.example:3]: Found it, it was the **Early access** app.

**Review by gerhard — APPROVED** [review]
👍

**Pull request merged**